user_guide/src/examples/df_manipulations/casting.py

```diff
@@ -0,0 +1,3 @@
+from .dataset import df
+
+df["a"] = df["a"].cast(float)
```

Comment:
Maybe mention that we can also cast with polars dtypes like `pl.Float32` or `pl.Float64`

Reply:
added as a TODO